src/scores/continuous/interval_impl.py

```diff
@@ -0,0 +1,100 @@
+"""
+Implementation of quantile interval score
+"""
+from typing import Optional
+
+import xarray as xr
+
+from scores.functions import apply_weights
+from scores.typing import FlexibleDimensionTypes
+from scores.utils import check_dims, gather_dimensions
+
+
+def quantile_interval_score(  # pylint: disable=R0914
+    fcst_lower_qtile: xr.DataArray,
+    fcst_upper_qtile: xr.DataArray,
+    obs: xr.DataArray,
+    lower_qtile_level: float,
+    upper_qtile_level: float,
+    *,  # Force keywords arguments to be keyword-only
+    reduce_dims: Optional[FlexibleDimensionTypes] = None,
+    preserve_dims: Optional[FlexibleDimensionTypes] = None,
+    weights: Optional[xr.DataArray] = None,
+) -> xr.Dataset:
+    """
+    Calculates the qauntile interval score for interval forecasts. This score penalizes
```

Comment:
Adding the mathjax would be very helpful

Reply:
I has now been added.